stdlib/public/Concurrency/AsyncThrowingStream.swift

```diff
@@ -189,11 +429,17 @@ extension AsyncThrowingStream.Continuation {
   }
 
   /// Resume the task awaiting the next iteration point by having it return
-  /// normally from its suspension point or buffer the value if no awaiting
-  /// next iteration is active where the `Element` is `Void`.
+  /// nomally from its suspension point.
+  ///
+  /// - Returns: A `YieldResult` indicating the success or failure of the
+  ///   yield operation.
+  ///
+  /// Use this method with `AsyncThrowingStream` instances whose `Element`
+  /// type is `Void`. In this case, the `yield()` call simply unblocks the
```

Comment:
```suggestion
  /// type is `Void`. In this case, the `yield()` call unblocks the
```

You don't need the adverb.